<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>322</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>322</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="HoleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hole</string>
   </property>
  </widget>
  <widget class="QLabel" name="PegPartLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Part:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PegFaceLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Face:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PegFaceIDField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FaceID</string>
   </property>
  </widget>
  <widget class="QLabel" name="TaskLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Label</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TaskLabelField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="AnimateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Animate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Get from selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="HoleFaceIDField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FaceID</string>
   </property>
  </widget>
  <widget class="QLabel" name="HolePartLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Part:</string>
   </property>
  </widget>
  <widget class="QLabel" name="HoleFaceLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Face:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PegLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Screw</string>
   </property>
  </widget>
  <widget class="QLabel" name="HolePartField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PartLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="PegPartField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PartLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ScrewTypeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ScrewTypeField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ThreadType</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
